docs/OPTIONS-LIST.md

```diff
@@ -133,6 +133,17 @@ for example `--ogspv LeelaZero`
 
 **IMPORTANT: see [notes H](/docs/NOTES.md#h) for details**
 
+### aichat
+
+`--aichat` Allow bots to send messages to the chat and malkovich log.
+
+Bots should output messages to standard error output in the format:
+
+```
```

Comment:
you need to add the flag `Text`, mardownlint enforces to always have a language flag for each code quotations blocks